### DISCUSSION TITLE
Manual flight in rate mode

### DISCUSSION THREAD

**Discussion by AsiaShi** [bitcraze/discussions#495] [Q&A]
I want to know how to use rate mode in manual flight，CF client has the option of rate control, but it is gray and unselectable，If you know how to set it up, please let me know. I would be grateful！
![Uploading Cf client.png…]()

**Comment by knmcguire** [comment]
Hi! 

I can not find this option for rate control on the cfclient tab... Where did you find this? Could you make a screen shot? Perhaps you have an older version of the cfclient.

**Reply by AsiaShi** [reply]
Maybe I have an older version of the cfclient，but how can i use rate mode in manual flight ? I want to adjust the inner loop PID of  angular rate.

**Comment by knmcguire** [comment]
The old version of the cfclient was pointing to an old non existing parameter called `flightmode.ratepid` that was removed a long time ago. So the button was never functional, so we removed the button 2 years ago with this commit: https://github.com/bitcraze/crazyflie-clients-python/commit/b9380d1391c80c6529b8cb84558ea7bbfaac403c

The [parameters ](https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/api/params/#flightmode) that you are looking for are probably flightmode.stabeModeRoll/Pitch/Yaw. These need to be set to 0 which is rate.

Also, please update the cfclient with `pip3 install cfclient --upgrade`

**Comment by AsiaShi** [comment]
Thank you！I have solved the problem，now i can use the rate mode to flight the crazyflie. Thanks again！

**Reply by knmcguire** [reply]
Did my above answer solve the problem? If so, could you mark it as an answer?
